src/CachePlugin.php

```diff
@@ -58,6 +59,28 @@ public function __construct(CacheItemPoolInterface $pool, StreamFactory $streamF
         $this->config = $optionsResolver->resolve($config);
     }
 
+    public static function clientCache(CacheItemPoolInterface $pool, StreamFactory $streamFactory, array $config = [])
```

Comment:
lets add a phpdoc here that says this sets up the cache plugin in a client mode. it will cache even with cache-control private or no-store.